pkg/resources/statefulsets/kes-statefulset.go

```diff
@@ -119,29 +124,50 @@ func NewForKES(t *miniov1.Tenant, serviceName string) *appsv1.StatefulSet {
 		}
 	}
 
+	if t.KESClientCert() {
+		clientCertSecret = t.Spec.KES.ClientCertSecret.Name
+	}
+
+	var volumeProjections []corev1.VolumeProjection
+
+	if t.Spec.KES.Configuration.Name != "" {
+		volumeProjections = append(volumeProjections, corev1.VolumeProjection{
+			Secret: &corev1.SecretProjection{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: t.Spec.KES.Configuration.Name,
+				},
+				Items: configPath,
+			},
+		})
+	}
+
+	if serverCertSecret != "" {
+		volumeProjections = append(volumeProjections, corev1.VolumeProjection{
+			Secret: &corev1.SecretProjection{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: serverCertSecret,
+				},
+				Items: serverCertPaths,
+			},
+		})
+	}
+
+	if clientCertSecret != "" {
+		volumeProjections = append(volumeProjections, corev1.VolumeProjection{
+			Secret: &corev1.SecretProjection{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: clientCertSecret,
+				},
+			},
```

Comment:
Shouldn't there be an `Items` field with path to mount here?

Reply:
In this case is not need it, since the user can provide all the certificates he needs, eg: `public.crt`, `private.key` or `ca.crt` or both, and then use them in the `server-config.yaml` he also provides.